sites/kit.svelte.dev/src/routes/+page.svelte

```diff
@@ -15,6 +15,112 @@
 	<meta name="twitter:title" content="SvelteKit" />
 	<meta name="twitter:description" content="Web development, streamlined" />
 	<meta name="description" content="SvelteKit is the official Svelte application framework" />
+
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="SvelteKit • Web development, streamlined" />
+	<meta property="og:description" content="SvelteKit is the official Svelte application framework" />
+	<meta property="og:url" content="https://kit.svelte.dev/" />
+	<meta property="og:image" content="https://raw.githubusercontent.com/sveltejs/branding/master/svelte-logo.svg" />
+
+	<script type="application/ld+json">
+	{
+		"@context": "http://schema.org",
+		"@type": "SoftwareSourceCode",
+		"name": "SvelteKit",
+		"description": "SvelteKit is the official Svelte application framework.",
+		"url": "https://kit.svelte.dev/",
+		"codeRepository": "https://github.com/sveltejs/kit",
+		"dateCreated": "2020-10-15T16:00:00",
+		"programmingLanguage": [
+			"JavaScript",
+			"TypeScript"
+		],
+		"logo": "https://raw.githubusercontent.com/sveltejs/branding/master/svelte-logo.svg",
+		"sameAs": [
+			"https://github.com/sveltejs/kit",
+			"https://www.npmjs.com/package/@sveltejs/kit"
+		],
+		"license": "https://github.com/sveltejs/kit/blob/master/LICENSE",
+		"releasedEvent": {
+			"@type": "PublicationEvent",
+			"startDate": "2022-12-14T17:00:00",
+			"url": [
+				"https://svelte.dev/blog/announcing-sveltekit-1.0",
+				"https://www.youtube.com/watch?v=N4BRVkQVoMc"
+			]
+		},
+		"contributor": [{
+			"@type": "Person",
+			"name": "Rich Harris",
+			"url": "https://www.benmccann.com",
```

Comment:
I think Rich's site is at https://www.rich-harris.co.uk (https://github.com/Rich-Harris/Rich-Harris.github.io), looks like it hasn't been updated in a while though 😅 

```suggestion
			"url": "https://www.rich-harris.co.uk",
```